.travis.yml

```diff
@@ -48,6 +48,7 @@ matrix:
   - compiler: clang
     osx_image: xcode6.4
     os: osx
+    cache:
```

Comment:
Because OS X is not on AWS the caching seems to not help. In a prior build, downloading the cache took over 3 minutes